tox.ini

```diff
@@ -1,16 +1,52 @@
 [tox]
-envlist = py27, py33, py34, py35, pypy, jython, pylint
+envlist = coverage-erase, py27, py33, py34, py35, pypy, jython, pylint, coveralls
```

Comment:
I would prefer coverage-erase and coveralls to not be part of the default testing procedure, but being run from travis.yml instead, because they need special setup on local machines (providing a token). Plus running the tests should trigger a coveralls call.